Cargo.toml

```diff
@@ -41,9 +42,11 @@ serde_json = "1.0"
 sha3 = "0.10"
 thiserror = "1.0"
 zeroize = "1.5"
+
 # Optional ones
 js-sys = { version = "0.3", optional = true }
 lazy_static = { version = "1.4", optional = true }
+pqc_kyber = { git = "https://github.com/tbrezot/kyber.git", branch = "feat/expose_indcpa_primitives", features = ["wasm", "ind_cpa"], optional = true}
```

Comment:
Did you open a PR on the main project about that?

Reply:
we won't be able to publish the crate with a `git` dependency, it does matter for KMS

Reply:
I didn't open a PR, but an issue before forking. I guess I could open a PR now.